api/HTMLElement.json

```diff
@@ -1848,20 +1848,36 @@
             "chrome_android": {
               "version_added": true
             },
-            "edge": {
-              "version_added": true
-            },
-            "edge_mobile": {
-              "version_added": true
-            },
+            "edge": [
+              {
+                "version_added": true,
+                "partial_implementation": true,
+                "notes": "Returns incorrect value for elements without an explicit tabindex attribute. See <a href='https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/4365703/'>issue 4365703</a> for details."
+              },
+              {
+                "version_added": "18"
```

Comment:
AFAIK the newer entry typical goes first.

Reply:
done